composer.json

```diff
@@ -16,10 +16,14 @@
     ],
     "require": {
         "composer/installers": "^1.2",
+        "drupal/bootstrap": "^3.21",
```

Comment:
This is no longer needed. Please `composer remove drupal/bootstrap` it.

Reply:
Done